main.go

```diff
@@ -82,10 +59,16 @@ func (*RootModule) NewModuleInstance(vu k6modules.VU) k6modules.Instance {
 // Exports returns the exports of the JS module so that it can be used in test
 // scripts.
 func (mi *ModuleInstance) Exports() k6modules.Exports {
-	return k6modules.Exports{Default: mi.mod}
+	return k6modules.Exports{
+		Named: map[string]interface{}{
+			"chromium": mi.mod,
+			"devices":  mi.mod.Devices,
+		},
```

Comment:
The JS API is better now :+1:, but we'll need to change this once we add support for other browsers. Probably by exporting each browser's `BrowserType` implementation.